.github/workflows/release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: |
           make release-notes
       - name: Release
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@4716bde57e0fcda6ce83002e2469b8a90e560573 # v1 https://api.github.com/repos/softprops/action-gh-release/git/tags/4716bde57e0fcda6ce83002e2469b8a90e560573
```

Comment:
v1 seems to be:
https://github.com/softprops/action-gh-release/releases/tag/v1 => 1e07f4398721186383de40550babbdf2b84acfc5

Reply:
I couldn't find 4716bde57e0fcda6ce83002e2469b8a90e560573

Reply:
Seems to point to this commit if I'm reading the api endpoint right:
https://github.com/softprops/action-gh-release/commit/1e07f4398721186383de40550babbdf2b84acfc5

API endpoint (from abolve) https://api.github.com/repos/softprops/action-gh-release/git/tags/4716bde57e0fcda6ce83002e2469b8a90e560573

I'm not sure why the shas are different though.

Reply:
> I couldn't find 4716bde57e0fcda6ce83002e2469b8a90e560573

Have you had a chance to look at this URL https://api.github.com/repos/softprops/action-gh-release/git/tags/4716bde57e0fcda6ce83002e2469b8a90e560573 ?

Reply:
Hm maybe some GitHub issue. The URL didn't work for me a few hours before. But v1 is definitely not pointing to this commit: https://github.com/softprops/action-gh-release/releases/tag/v1

Reply:
Ah I remember this doesn't work: https://github.com/softprops/action-gh-release/commit/4716bde57e0fcda6ce83002e2469b8a90e560573

Usually this works for all commits existing in a repo. Maybe I did something wrong

Reply:
Ah I now I understand. This URL is pointing to the `4716bde57e0fcda6ce83002e2469b8a90e560573` tag, not the `4716bde57e0fcda6ce83002e2469b8a90e560573` commit. Why are we using that tag?

Reply:
I wrote something up https://gist.github.com/naveensrinivasan/ca008c07279176acce28969fb77d056f hope this helps.

Reply:
Okay got the API query issue. But couldn't we just use the commit-id linked at the release in the GitHub UI?
=> https://github.com/softprops/action-gh-release/releases/tag/v1 (https://github.com/softprops/action-gh-release/commit/1e07f4398721186383de40550babbdf2b84acfc5)

Reply:
But I think the most important question is, what would dependabot do? 

Does it update to the latest release **or** the latest tag and which sha would it use then? (and also would it update the URL comment, because I prefer not adding the URL if we would have to update it manually after a dependabot PR)

Reply:
It wouldn't add the URL. The dependabot would update to the latest tag instead of the version.

Reply:
> It wouldn't add the URL
Okay then let's drop the URLs

Would it update the comment below the sha?

> update to the latest tag instead of the version
To clarify, the latest tag instead of the latest release? (or what do you mean with version?)